### ISSUE TITLE
Add radian to Homebrew repository

### ISSUE THREAD

**Issue by kalaschnik** [randy3k/radian#350]
Are there any plans to add radian to the homebrew repo?

**Comment by randy3k** [comment]
Why it is needed? radian is just a python package. Your could install python from homebrew and use its pip to install radian.

**Comment by kalaschnik** [comment]
That is correct! I was just tinkering around with my dotfiles, and wanted to enque radian in my `Brewfile`. But since I already have shell file for global npm packages, I better make one for global python packages and put `radian` there.
